docs/split-testing.md

```diff
@@ -0,0 +1,189 @@
+# Setting up a split testing experiment
+
+One thing we like to do from time to time is test changes
+on specific groups of pages and see how they perform.
+We call this "split testing".
+In contrast to traditional A/B testing,
+where the change being tested is shown to a portion of the _audience_,
+in the case of split testing,
+the change is shown to everyone, but only on a subset of similar pages
+(e.g., a group of related Ask CFPB answers).
+
+
+## So you've been asked to set up a split testing experiment
+
+Good news! The infrastructure has been put in place for you to do it easily.
+It's set up so that the code being tested is isolated by a feature flag.
+
+### The flag condition
+
+In order to check whether or not a page is part of a split testing cluster,
+we registered a new flag condition in `cfgov/core/feature_flags.py`.
+
+```python
+@conditions.register('in split testing cluster')
+def in_split_testing_cluster(cluster_group, page, clusters=CLUSTERS, **kwargs):
+    cluster_group = clusters[cluster_group]
+    lookup_value = getattr(page, 'split_test_id', page.id)
+    return lookup_value in chain(*cluster_group.values())
+```
```

Comment:
Is it necessary to include the full code for the condition if someone who is setting up a split testing cluster doesn't need to modify it? They just need to know it exists and how to use it, right?